eng/common/docgeneration/templates/matthews/styles/main.js

```diff
@@ -161,7 +161,7 @@ function populateIndexList(selector, packageName) {
     httpGetLatestAsync(latestGAUrl, latestVersions, packageName)
     httpGetLatestAsync(latestPreviewUrl, latestVersions, packageName)
     var publishedVersions = $('<ul style="display: none;"></ul>')
-    var collapsible = $('<div class="versionarrow">&nbsp;&nbsp;&nbsp;Other versions</div>')
+    var collapsible = $('<div class="versionarrow">&nbsp;&nbsp;&nbsp;All versions</div>')
```

Comment:
How hard would it be to hide the latest ga/latest preview versions when someone expands the all versions section? Ideally it would be good to come up with a way to not list any versions twice.

Reply:
It asks a filter logic. Not too hard, but it takes several secs to load all versions to the page. Might take a little extra time. But if this is ideal case, I can fix it like what you said.

Reply:
The other option would be to exclude the ga/latest from the "Other list" I'm not sure which is easier.